package.json

```diff
@@ -3,12 +3,13 @@
   "version": "0.0.0",
   "scripts": {
     "clean:game": "rm -r saves/*.json",
-    "compile:gameStateSchema": "json2ts src/utils/types/gameState.schema.json > src/utils/types/gameState.d.ts && ts-node scripts/compileRoomTypesTuple.ts",
+    "compile:schemas": "ts-node scripts/denormalizeSchemas.ts && json2ts --cwd=src/utils/schemas/normalized/ src/utils/schemas/normalized/gameState.json > src/utils/types/gameState.d.ts && ts-node scripts/compileRoomTypesTuple.ts && prettier --loglevel silent --write src/utils/types/gameState.d.ts src/utils/types/roomTypesTuple.ts",
```

Comment:
Should the `prettier` stuff be it's own command that `compile:schema` calls at the end? That way `compile:schema` doesn't get too long.

Reply:
I was thinking of breaking `compile:schema` into several commands.

- denormalize schemas
- schema to typescript
- compile room types tuple
- prettier

I wasn't sure what to call each command

Reply:
updated